api/db/changelog/db.changelog-218-drop-unused-columns-user-table.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="thibault" id="db.changelog-218-drop-unused-columns-user-table">
+    <dropColumn columnName="compliance_training_bypass_time" tableName="user"/>
+    <dropColumn columnName="compliance_training_completion_time" tableName="user"/>
+    <dropColumn columnName="compliance_training_expiration_time" tableName="user"/>
```

Comment:
We don't store expiration times any more - instead, we calculate them based on completion time and the annual renewal logic.  See `AccessModuleServiceImpl.getExpirationTime()`